src/index.js

```diff
@@ -2,8 +2,20 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import routes from './routes'
 import './index.css'
+import Home from './containers/Home/index'
```

Comment:
`index` could be omitted, `import Home from './containers/Home'`

Reply:
This part can be delete. Not be used.

Reply:
yep, deleted!